content/docs/0.8.x/reference/cli/_index.md

```diff
@@ -198,3 +198,144 @@ The `KEPTNCONFIG` environment variable holds a config file that contains the cre
 * If `KEPTNCONFIG` environment variable is defined, the Keptn CLI reads the credentials from it. 
 
 * If the credentials are not found in the config file, it will fall back to the credential manager.
+
+## Enabling shell autocompetion of Keptn CLI
+
+Keptn CLI provides autocompletion support for Bash and Zsh, which can save you a lot of typing.
+
+Below are the procedures to set up autocompletion for Bash and Zsh. Please expand the corresponding section matching your configuration.
+
+<details><summary>Autocompletion for Bash (Linux)</summary>
+
+The Keptn CLI completion script for Bash can be generated with the command `keptn completion bash`. Sourcing the completion script in your shell enables Keptn CLI autocompletion.
+
+However, the completion script depends on bash-completion, which means that you have to install this software first (you can test if you have bash-completion already installed by running type _init_completion).
```

Comment:
Please change type _init_completion to `type _init_completion` (add single quotes)